azurerm/internal/services/kusto/kusto_iothub_data_connection_resource.go

```diff
@@ -76,6 +76,44 @@ func resourceKustoIotHubDataConnection() *pluginsdk.Resource {
 				ValidateFunc: iothubValidate.IoTHubConsumerGroupName,
 			},
 
+			"table_name": {
```

Comment:
We try to order the schema so that Required fields are described first, followed by any Optional fields; could you add these right after `shared_access_policy_name`?

Reply:
Good point, I missed that.